src/string_bitflip.cc

```diff
@@ -0,0 +1,20 @@
+#include "string_bitflip.h"
+
+#include "node.h"
+#include "v8.h"
+
+#include <string.h>
+#include <limits.h>
+
+namespace node {
+
+void StringBitflip::BE_FlipBits (uint16_t* dst,
```

Comment:
Style: no space before the paren.  The function name is a little off style-wise.  I'd name it `SwapBytes()` without the underscore.